src/extension.ts

```diff
@@ -286,9 +286,8 @@ export function activate(context: vscode.ExtensionContext): void {
     registerCommand('appService.DisableRemoteDebug', async (node?: SiteTreeItem) => disableRemoteDebug(node));
 
     registerCommand('appService.showFile', async (node: FileTreeItem) => {
-        const logFiles: string = 'LogFiles/';
         // we don't want to let users save log files, so rather than using the FileEditor, just open an untitled document
-        if (node.path.startsWith(logFiles)) {
+        if (node.path.toLowerCase().match(/logfiles(\/|\\)/g)) {
```

Comment:
- Use `^` to make sure this is at the start of the string (Right? `otherFolder\LogFiles\fdsafasd` shouldn't count as logs right?)

nits:
- Use the 'i' flag to indicate "case insensitive" instead of ".toLowerCase" stuff
- Use ".test" instead of ".match" which actually returns a boolean instead of match array
- Don't use 'g' flag (g indicates "global" which means "don't return after first match" which we don't care about here)

```
if (/^LogFiles(\/|\\)/i.test(node.path))
```